core/templates/pages/exploration-editor-page/services/exploration-diff.service.ts

```diff
@@ -264,10 +320,9 @@ export class ExplorationDiffService {
    *  - 'target': target state of link
    *  - 'linkProperty': 'added', 'deleted' or 'unchanged'
    */
-  // TODO(#7176): Replace 'any' with the exact type.
   _compareLinks(
-      v1States: any, originalStateIds: Array<string>,
-      v2States: any, newestStateIds: Array<string>): Array<any> {
+      v1States: IStateObjectsDict, originalStateIds: IStateIds,
+      v2States: IStateObjectsDict, newestStateIds: IStateIds): IStateLink[] {
```

Comment:
each argument on one line please

Reply:
Ditto in other places, except when the args fit on one line.

Reply:
done